shared/constants/teams.js

```diff
@@ -21,6 +21,17 @@ export type CreateNewTeamFromConversation = NoErrorTypedAction<
   }
 >
 
+export type AddToTeam = NoErrorTypedAction<
+  'teams:addToTeam',
+  {
+    name: string,
+    email: string,
+    username: string,
+    role: null | 'reader' | 'writer' | 'admin' | 'owner',
```

Comment:
should probably make this its own type without the null, e.g. `export type TeamRole = 'reader' | ... | 'owner'` and have `TeamRole` or `?TeamRole` as necessary.

Also, does it have to be nullable? Seems like we should know the role if we're going to add to a team

Reply:
Good point; it only really needs to be nullable at the UI level, thanks for the suggestion